### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by crosszay1** [pybricks/support#1815] [Q&A]
I'm completely new to pybricks and heres the story: 
I installed pybricks and it seems to have installed but the "home" button on the hub keeps flashing blue. bluetooth does not seem to be working via pybricks. in addition to that it seems my computer no longer recognizes the hub as a usb device. I'm pretty worried I may have just bricked (haha) my hub. some help would be great

**Comment by dlech** [comment]
The blinking Bluet light means that it is advertising via Bluetooth, so you can click the Bluetooth button in Pybricks Code (https://code.pybricks.com) and you should be able to connect to your hub - assuming you have a compatible browser and a compatible Bluetooth adapter.

If you ever want to go back to the official LEGO firmware, just follow the directions here: https://pybricks.com/learn/getting-started/install-pybricks/#restoring-the-original-firmware
